app/controllers/tools_controller.rb

```diff
@@ -37,16 +36,8 @@ def set_body_id
     @body_id = 'tools'
   end
 
-  def tools
-    tool_class.order(published_at: :desc).live.published
-  end
-
-  def set_featured_tools
-    @featured_tools = tools.english.where.not(buy_url: nil).map { |tool| tool if tool.buy_url.present? }.compact
-  end
-
   def set_tools
-    @tools = tools.english.map { |tool| tool if tool.buy_url.blank? }.compact
+    @tools = tool_class.reorder(position: :desc).order(published_at: :desc).live.published
```

Comment:
```suggestion
    @tools = tool_class.english.reorder(position: :desc).order(published_at: :desc).live.published
```

this is need for the `preferred_localization` to work